crate/README.MD

```diff
@@ -0,0 +1,73 @@
+# CrateDB Helm Chart
+
+This directory contains a Kubernetes chart to deploy a [CrateDB](https://crate.io) cluster using a StatefulSet.
+
+## Prerequisites Details
+* Kubernetes 1.5+
+* PV support on the underlying infrastructure
+
+## StatefulSet Details
+* https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/
+
+## StatefulSet Caveats
+* https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#limitations
+
+## Todo
+* 
+
+## Chart Details
+This chart will do the following:
+
+* Implement a HA scalable CrateDB cluster using a Kubernetes StatefulSet.
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```console
+$ helm repo add oc https://orchestracities.github.io/charts/
+$ helm dependency update
+$ helm install --name my-release oc/crate
+```
+
+## Connecting to CrateDB
+
```

Comment:
what's planned here? something like port-forward instructions or execs into pod maybe?

Reply:
yes!

Reply:
ok, maybe we don't need to write this for all charts in the readme. There was a part in the templates where this was actually written so it'd be printed once the chart is deployed. Less text to maintain I mean.